src/config.coffee

```diff
@@ -12,3 +13,15 @@ TODO rename
 @op_list = "opList" # var name
 @fix_underscore = "fx" # prefix var name
 @reserved = "reserved" # prefix var name
+
+@int_type_list = ["int"]
```

Comment:
This lists seems unused. It is intended?

Reply:
see type_inference (long diff collapsed)